### PR TITLE
ceph_disk: remove unused function parameter

### PR DESCRIPTION
**Pull request by bellaalleb**
Signed-off-by: Jing Li <lijing@gohighsec.com>

### CODE REVIEW

**Comment by wjwithagen** [comment]
This is to allow to perpare a series of disks.
Why not fix the commandline interpretation and allow ceph-disk to take more disks on the commandline?

**Comment by bellaalleb** [comment]
@wjwithagen This patch only remove an unused parameter. The function "prepare_disk" (line 2988, main.py) takes *to_prepare_list and but didn't use it

**Comment by wjwithagen** [comment]
@bellaalleb 
I still wonder wether it is not wiser to allow a series of disk to be zapped and use _to_prepare for that, and fill the list while parsing for zap args.
But I guess yours will work due to zap using `self.data.args`

File: qa/workunits/ceph-disk/ceph-disk-no-lockbox [hunk 1]
```diff
@@ -2310,7 +2310,7 @@ class PrepareData(object):
         if self.args.osd_uuid is None:
             self.args.osd_uuid = str(uuid.uuid4())
 
-    def prepare_device(self, *to_prepare_list):
+    def prepare_device(self):
```
Comment (tchaikov) [inline qa/workunits/ceph-disk/ceph-disk-no-lockbox]: yes, technically, this parameter is not used at all. but IMHO, `PrepareData.prepare_device()` is not just a utility method used by its children classes. it also act as an interface documenting what a derived class should implement.

**Comment by tchaikov** [comment]
as ceph-disk will be deprecated. only bug fixes are accepted now. closing this now. if you think otherwise, please feel free to reopen it.

**Pull request closed without merging**